paper-server/patches/sources/com/mojang/brigadier/tree/CommandNode.java.patch

```diff
@@ -13,6 +13,7 @@
 +    public CommandNode<net.minecraft.commands.CommandSourceStack> clientNode; // Paper - Brigadier API
 +    public CommandNode<io.papermc.paper.command.brigadier.CommandSourceStack> unwrappedCached = null; // Paper - Brigadier Command API
 +    public CommandNode<io.papermc.paper.command.brigadier.CommandSourceStack> wrappedCached = null; // Paper - Brigadier Command API
++    public org.bukkit.command.Command wrappedBukkitCommandCached = null; // Paper - Brigadier Command API
```

Comment:
Semantics but, I think cachedWrappedBukkitCommand would read better than what's currently there

Reply:
Yeah, makes sense to me. I think I originally went with `wrappedBukkitCommand` + `Cached` to match the naming of `unwrapped` + `Cached` and `wrapped` + `Cache` directly above.

Maybe Owen had a reason to go with the original naming, but in isolation I prefer `cached` + `WrappedBukkitCommand`.